### DISCUSSION TITLE
[Feature Request] (VAutocomplete) add 'column-count' css to item-props

### DISCUSSION THREAD

**Discussion by patrickkuo111** [vuetifyjs/vuetify#16717] [General]
### Problem to solve
I hava a long list of data to put into 'v-autocomplete', and it is too long for scrollbar to roll. I want to display data in 3 columns. 

like below: 
![image info](https://i.ibb.co/XWk5WbS/001.png)

### Proposed solution
I found out to add 'column-count: 3' in style will solve that. But I could not find the way to add this style in VAutocomplete's v-list

**Comment by KaelWD** [comment]
https://vuetifyjs.com/en/api/v-autocomplete/#props-menu-props
https://vuetifyjs.com/en/api/v-menu/#props-content-class
https://vuejs.org/api/sfc-css-features.html#deep-selectors

**Reply by patrickkuo111** [reply]
I did try <v-select :menu-props="{ contentClass: 'my-awesome-list' }" and ::v-deep {} , but somehow not working

**Reply by patrickkuo111** [reply]
Finally, I got it. For ppl whom needs it.

in css part: 
  .columnwrapper .v-list{
    column-count: 3;
  }

in v-autocomplete
    :menu-props="{
        closeOnClick:true,
        nudgeBottom: 15 + 'px',
        zIndex: 2,
        contentClass: 'columnwrapper',
      }"

note: still not sure how to do it in ::deep {}，I can not find the proper class name to override it.
